app/lib/formatter.rb

```diff
@@ -157,7 +153,12 @@ def encode_custom_emojis(html, emojis, animate = false)
         emoji     = emoji_map[shortcode]
 
         if emoji
-          replacement = "<img draggable=\"false\" class=\"emojione\" alt=\":#{encode(shortcode)}:\" title=\":#{encode(shortcode)}:\" src=\"#{encode(emoji)}\" />"
+          original_url, static_url = emoji
+          replacement = if animate
```

Comment:
I would use begin/end to make this assignment more readable, as I've been doing recently.

Reply:
Done